daemon/main.go

```diff
@@ -186,6 +186,7 @@ func init() {
 	flags.StringVar(&v4Prefix, "ipv4-range", "", "IPv4 prefix")
 	flags.StringVarP(&config.Tunnel, "tunnel", "t", "vxlan", "Tunnel mode vxlan or geneve, vxlan is the default")
 	flags.StringVar(&bpfRoot, "bpf-root", "", "Path to mounted BPF filesystem")
+	flags.String("access-log", "", "File to store access log in")
```

Comment:
if lf := viper.GetString("access-log"); lf != "" {
                          if logFile, err = os.Create(lf); err != nil {

the value given is the path to the file. I'll change the wording of the usage string, that seems to be confusing.